src/scancode/cli.py

```diff
@@ -458,11 +465,19 @@ def scancode(ctx,
             full_root=full_root,
             ignore=user_ignore)
 
+        for option, process in plugincode.post_scan.get_post_scan_plugins().items():
+            if kwargs[option.replace('-', '_')]:
+                options['--' + option] = True
+                if not quiet:
+                    echo_stderr('Running post-scan plugin: %(option)s...' % locals(), fg='green')
+                results = process(scanners, results, options, input)
```

Comment:
Also, I'm not sure what you mean by *calling* the `scanners`, I mean `scanners` is just passed as a param which might or might not be used by a `post-scan` plugin.
e.g `only-findings` uses it, whereas `mark-source` doesn't